src/shared/ui/form/InputField.tsx

```diff
@@ -3,25 +3,31 @@ import { ColorValue, TextInputProps } from 'react-native';
 
 import { Controller, useFormContext, useController } from 'react-hook-form';
 
-import { Input } from '@shared/ui';
+import { Box, Input, XStack } from '@shared/ui';
 import { ErrorMessage } from '@shared/ui/form';
 
-type Props = {
+export type InputFieldProps = {
```

Comment:
That's actually optional but makes sense to do as we don't want to expose unnecessary types because there are already too many of them. Since it is not used anywhere and most likely will not be used let's not export it.
Also, we usually name it just `Props` (if it's not exported) because, in our opinion, it is easier to locate them when one is looking at the file. Or maybe it is just the code style of choice :)

```suggestion
type Props = {
```

Reply:
My bad, I thought I returned it back to how it was before.